test/cell_test.rb

```diff
@@ -59,5 +59,17 @@ def test_fire_upon_lowers_ship_health_by_one
     @cell.place_ship(@submarine)
     @cell.fire_upon
     assert_equal 1, @cell.ship.health
+    assert_equal 2, @cell.ship.health
+  end
+
+  def test_cell_has_not_been_fired_upon
+    assert_equal false, @cell.fired_upon?
+    @cell.fire_upon
+    assert_equal true, @cell.fired_upon?
+  end
+
+  def test_fire_upon_reduces_health
```

Comment:
I think we need to add a line at the beginning of this test where we invoke @cell.place_ship so that in line 73 there is an @ship in the cell that gets hit -- specifically the first ship, "Cruiser", which is stored in the @ship instance variable.